docs/calendar.md

```diff
@@ -36,6 +36,46 @@ const calendar = new Intl.DateTimeFormat().resolvedOptions().calendar;
 date.withCalendar(calendar).add({ months: 1 });
 ```
 
+### Invariants Across Calendars
+
+The following "invariants" (statements that are always true) hold for all built-in calendars, and SHOULD hold for properly-authored custom calendars:
+
+- `year` is always an integer that increases as time goes forward
+- `month` and `day` are always positive integers that increase as time goes forward, except they reset at the boundary of a year or month, respectively
+- `date.month === 1` during the first month of any year
+- `date.month === date.monthsInYear` during the last month of any year
+- `month` is always continuous (no gaps)
+- Any date can be serialized to an object using only four properties: `{ year, month, day, calendar }`
```

Comment:
I wouldn't make this claim, because we can't guarantee this to be true for custom calendars.  The ISO fields are the things guaranteed to round-trip.

Reply:
I think it's claimed to be the case for builtin calendars, and recommended for custom calendars - see the language on line 41.

Reply:
I don't know though if I'd go so far to claim this invariant SHOULD hold for custom calendars.  This works so long as custom calendars use something that roughly maps to years, months, and days.  But a custom calendar could use wholly different units.  We often use shamrocks, rainbows, and leprechauns as an example.

Reply:
OK I checked in new text above.